### PR TITLE
Add option to passthrough the http traffic to specified list of servers.

### PR DESCRIPTION
**Pull request by kowalski**

### CODE REVIEW

File: tests/test_aioresponses.py [hunk 1]
```diff
@@ -118,3 +118,12 @@ def test_multiple_requests(self):
             self.assertEqual(resp.status, 201)
             resp = self.loop.run_until_complete(self.session.get(self.url))
             self.assertEqual(resp.status, 202)
+
+    def test_passthrough(self):
+        self.session._request = mocked = Mock()
```
Comment (pnuckowski) [inline tests/test_aioresponses.py]: Maybe it would be better to "hit" actual server like example.com :) ?

**Comment by kowalski** [comment]
Well.... example.com doesnt exist so whatever server we want to hit we have
to spin it up first in the test setup...

I didnt want to change too much with this feature l...



30.01.2017 09:58 "pnuckowski" <notifications@github.com> napisał(a):

**Comment by pnuckowski** [comment]
@kowalski Can you resolve conflict ? After it will be ready to merge.

**Comment by kowalski** [comment]
@pnuckowski I've rebased this branch

**Pull request merged**